frontend/src/components/reports/ReportingHistoryTab.vue

```diff
@@ -1,90 +1,134 @@
 <template>
-  <v-container fluid class="pa-0">
-    <v-row class="mt-1 mb-4">
+  <v-container fluid class="pa-0 ma-0">
+    <ReportingSearchCard :loading="loading" class="mb-6" @search="loadSubmittedReports" />
+    <v-row no-gutters class="my-4">
       <v-col cols="12" md="5">
         <h2>Report Details</h2>
       </v-col>
       <v-col cols="12" md="7">
-        <FacilityFilter :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
+        <FacilityFilter v-if="!isEmpty(submittedReports)" :loading="loading" :default-show-input="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
       </v-col>
     </v-row>
-
-    <v-data-table :headers="headers" :items="filteredSubmittedReports" item-key="surveyResponseId" density="compact">
-      <template #[`item.status`]="{ item }">
-        <span :class="getStatusClass(item.status)" class="px-2 py-1">{{ item.status }}</span>
-      </template>
-      <template #[`item.actions`]="{ item }">
-        <v-btn icon="mdi-eye-outline" variant="text" @click="viewSurveyResponse(item)" />
-        <v-btn icon="mdi-tray-arrow-down" variant="text" @click="false" />
-      </template>
-    </v-data-table>
+    <v-skeleton-loader :loading="loading" type="table-tbody">
+      <AppAlertBanner v-if="isEmpty(submittedReports)" type="info" class="mt-4">
+        <div>You have no submitted reports.</div>
+      </AppAlertBanner>
+      <v-data-table v-else :headers="headers" :items="filteredSubmittedReports" item-key="surveyResponseId" density="compact" :mobile="null" mobile-breakpoint="md" class="soft-outline">
+        <template #[`item.title`]="{ item }">
+          <span>{{ getReportTitle(item) }}</span>
+        </template>
+        <template #[`item.status`]="{ item }">
+          <span :class="getStatusClass(item)">{{ getStatusText(item) }}</span>
+        </template>
+        <template #[`item.submittedDate`]="{ item }">
+          {{ format.formatDate(item?.submittedDate) }}
+        </template>
+        <template #[`item.actions`]="{ item }">
+          <v-btn icon="mdi-eye-outline" variant="text" @click="viewSurveyResponse(item)" />
+          <v-btn icon="mdi-tray-arrow-down" variant="text" @click="false" />
```

Comment:
I think there's some desire to move away from icons to buttons/text. Maybe check the latest wireframe.

Reply:
yes, it should be handled by this ticket: OFMCC-4146